pages/configuration/system-configuration.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: System configuration
+description: Explore the documentation page for Memgraph and get insights into system configuration options.
+---
+
+import { Callout } from 'nextra/components'
+import { Steps } from 'nextra/components'
+
+# System Configuration
+
+## Adjusting fs.file-max
+
+The `fs.file-max` parameter in Linux sets the maximum number of file handles
+that the Linux kernel will allocate. Adjusting this is crucial when the system
+needs to handle numerous open files simultaneously. To adjust this parameter do the following: 
+
+<Steps>
+
+### Open `/etc/sysctl.conf` in a text editor
```

Comment:
{<h3> TITLE </h3>}

I think I would use this for headings so they are not visible in the TOC